assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -429,16 +430,25 @@ const baseActions = {
 			);
 
 			const failedAudiencesToRetry = [];
+			let insufficientPermissionsError = null;
 
 			audienceCreationResults.forEach( ( result, index ) => {
 				const audienceSlug = audiencesToCreate[ index ];
+
 				if ( result.error ) {
+					if ( isInsufficientPermissionsError( result.error ) ) {
+						insufficientPermissionsError = result.error;
+					}
 					failedAudiencesToRetry.push( audienceSlug );
```

Comment:
We aren't going to retry the failed audiences if `insufficientPermissionsError` is set, so should wrap this in an `else` to be clearer. I'll apply this myself to save the round trip.

```suggestion
					} else {
						failedAudiencesToRetry.push( audienceSlug );
					}
```